### ISSUE TITLE
Renaming the exporter? What is your opinion?

### ISSUE THREAD

**Issue by mrlhansen** [mrlhansen/idrac_exporter#113]
**This issue is for getting _your_ feedback on _potentially_ renaming the project.**

There have been some requests for renaming the exporter, but I have been hesitant because it's impractical. Most notably, renaming the project will also mean renaming the Docker and Helm repository, which means that all installations using either will have the be migrated to the new repositories.

Even if I rename it, then there is the question of name. There are already several bmc and redfish exporters. I was thinking `redfish_bmc_exporter` which is unique from what I can see.

Yay or nay? Leave a comment below.

**Comment by AlexG14** [comment]
Nay.
I can understand reasons behind this requests but consequences are a bit more than replacing a few symbols in a helm chart\docker compose file. You will lose all references (forums\mailing lists). Yes, you always can point to the new project and leave this one 'frozen' forever. Anyone can use any name for container so there is a very limited benefit from renaming.

**Comment by ilanni2460** [comment]
> **This issue is for getting _your_ feedback on _potentially_ renaming the project.**
> 
> There have been some requests for renaming the exporter, but I have been hesitant because it's impractical. Most notably, renaming the project will also mean renaming the Docker and Helm repository, which means that all installations using either will have the be migrated to the new repositories.
> 
> Even if I rename it, then there is the question of name. There are already several bmc and redfish exporters. I was thinking `redfish_bmc_exporter` which is unique from what I can see.
> 
> Yay or nay? Leave a comment below.

I agree with your proposal. At first glance, IDRAC only supports Dell. In fact, it now supports data collection from many manufacturers.

**Comment by love6875520** [comment]
I think keep this name is better

**Comment by boxstep** [comment]
are there any plans to add more metrics? i get only about 100 returned, while other redfish_exporters return over 500. SNMP returns 2200+. Some metrics for various controller statuses are missing for example raid controller status

**Comment by mrlhansen** [comment]
@boxstep If there is a request for it and the metrics are useful in general, then yes. I am not adding every single ting that can possibly be exported as a metric. The goal is to have an exporter that exposes a uniform set of metrics for different systems. Your example with raid controller health could be added. If you want to discuss this further, please open a new issue.

**Comment by mmelicias** [comment]
If you decide in long time support others systems like Fujitsu 👍 Maybe renaming like redfish_exporter, but is your decision

**Comment by robotneo** [comment]
It is recommended to rename it and replace it with redfish_exporter, because other brands will definitely need to be added later.
